spec/features/admin_can_add_genre_on_index_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe 'As an admin user,' do
+  describe 'When I visit the genre index' do
+    it 'I can fill out a form to create a new genre' do
+      admin = User.create(username: 'funbucket13', password: 'test', role: 1)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+      visit genres_path
+
+      fill_in 'genre[name]', with: 'Action'
+      click_on 'Create Genre'
+
+      genre = Genre.last
+
+      expect(page).to have_content(genre.name)
```

Comment:
You generally don't want to call directly to the Model in a feature test (line 13). I'd rather see an assertion for "Action" here since we don't naturally have a genre object to call to.